### ISSUE TITLE
netcore 3 - Sync read disallowed exception

### ISSUE THREAD

**Issue by stephenlautier** [contentful/contentful.net#192]
After updating to netcore3 by default the contentful middleware is  throwing an error when its invoked.

#### Message
Synchronous operations are disallowed. Call ReadAsync or set AllowSynchronousIO to true instead.

#### Stacktrace
```
 at Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http.HttpRequestStream.Read(Byte[] buffer, Int32 offset, Int32 count)
   at System.IO.StreamReader.ReadBuffer()
   at System.IO.StreamReader.ReadToEnd()
   at Contentful.AspNetCore.MiddleWare.WebhookMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Builder.Extensions.MapMiddleware.Invoke(HttpContext context)
   at Odin.AspNet.Shared.Error.ErrorHandlingMiddleware.Invoke(HttpContext httpContext, RequestContext requestContext)
```

As i imagine its from the following line https://github.com/contentful/contentful.net/blob/master/Contentful.AspNetCore/MiddleWare/WebhookMiddleware.cs#L99

NOTE: the `AllowSynchronousIO` is by default on in netcore 3 (not something we configured

**Comment by Roblinde** [comment]
Thanks for this @stephenlautier I'll look into it and get it fixed in the next version.

**Comment by Roblinde** [comment]
@stephenlautier this should now be fixed in version 4.4.0. Thanks for reporting this!
